moped-database/migrations/1692145812500_set_fk_public_moped_proj_tags_tag_id/up.sql

```diff
@@ -0,0 +1,5 @@
+alter table "public"."moped_proj_tags" drop constraint "moped_proj_tags_tag_id_fkey",
+  add constraint "moped_proj_tags_tag_id_fkey"
+  foreign key ("tag_id")
+  references "public"."moped_tags"
+  ("id") on update cascade on delete cascade;
```

Comment:
this is consistent with our philosophy about cascades. if a dev is deleting from this table they must have a good reason to do so—so allow it.

Reply:
☝️ it's necessary for the other down migration to work.